EPlast/EPlast.Tests/Controllers/CityControllerTests.cs

```diff
@@ -0,0 +1,538 @@
+        }
+
+
+
+        [Test]
+        public async Task RemoveFollower_Valid_Test()
+        {
+            _cityMembersService.Setup(c => c.RemoveFollowerAsync(It.IsAny<int>()));
+
+            _logger.Setup(l => l.LogInformation(It.IsAny<string>()));
+
+            CitiesController citycon = CreateCityController;
+
+            var result = await citycon.RemoveFollower(GetFakeID());
+
+            Assert.NotNull(result);
+            Assert.IsInstanceOf<OkResult>(result);
+        }
+
+        [Test]
+        public async Task ChangeApproveStatus_Valid_Test()
+        {
+            _cityMembersService.Setup(c => c.ToggleApproveStatusAsync(It.IsAny<int>()))
+                .ReturnsAsync(new CityMembersDTO());
+
+            _logger.Setup(l => l.LogInformation(It.IsAny<string>()));
+
+            CitiesController citycon = CreateCityController;
+
+            var result = await citycon.ChangeApproveStatus(GetFakeID());
+
+            Assert.NotNull(result);
+            Assert.IsInstanceOf<OkObjectResult>(result);
+        }
+
+        [Test]
+        public async Task EditAdmin_Valid_Test()
+        {
+            CityAdministrationViewModel admin = new CityAdministrationViewModel();
+
+            _mapper.Setup(m => m.Map<CityAdministrationViewModel, CityAdministrationDTO>(It.IsAny<CityAdministrationViewModel>()))
+                .Returns(new CityAdministrationDTO());
+
+            _cityAdministrationService.Setup(c => c.EditAdministratorAsync(It.IsAny<CityAdministrationDTO>()));
+
+            _logger.Setup(l => l.LogInformation(It.IsAny<string>()));
+
+            CitiesController citycon = CreateCityController;
+
+            var result = await citycon.EditAdmin(admin);
+
+            Assert.NotNull(result);
+            Assert.IsInstanceOf<OkObjectResult>(result);
+        }
+
+        [Test]
+        public async Task RemoveAdmin_Valid_Test()
+        {
+
+
+            _mapper.Setup(m => m.Map<CityAdministrationViewModel, CityAdministrationDTO>(It.IsAny<CityAdministrationViewModel>()))
+                .Returns(new CityAdministrationDTO());
+
+            _cityAdministrationService.Setup(c => c.RemoveAdministratorAsync(It.IsAny<int>()));
+
+            _logger.Setup(l => l.LogInformation(It.IsAny<string>()));
+
+            CitiesController citycon = CreateCityController;
+
+            var result = await citycon.RemoveAdmin(GetFakeID());
+
+            Assert.NotNull(result);
+            Assert.IsInstanceOf<OkResult>(result);
+        }
+
+
+        [Test]
+        public async Task GetCitiesThatUserHasAccessTo_Valid_Test()
+        {
+
+            _cityAccessService.Setup(c => c.GetCitiesAsync(It.IsAny<ClaimsPrincipal>()));
+
+            CitiesController citycon = CreateCityController;
+
+            var result = await citycon.GetCitiesThatUserHasAccessTo();
+
+            Assert.NotNull(result);
+            Assert.IsInstanceOf<OkObjectResult>(result);
+        }
+
+
+       
+        public int GetFakeID()
```

Comment:
If you have faking function, use private modification
```suggestion
        private int GetFakeID()
```

Reply:
Changed it